hedera-mirror-rest/utils.js

```diff
@@ -1096,6 +1096,19 @@ const buildComparatorFilter = (name, filter) => {
   };
 };
 
+/**
+ * Calculates the expiryTimestamp.
+ * @param {int} autoRenewPeriod: seconds format
+ * @param {BigInt} createdTimestamp: nnnnnnnnnnnnnnnnnnn format format
+ * @param {BigInt} expirationTimestamp: nnnnnnnnnnnnnnnnnnn format format
+ * @returns {String} seconds.nnnnnnnnn format
+ */
+const calculateExpiryTimestamp = (autoRenewPeriod, createdTimestamp, expirationTimestamp) => {
+  return _.isNil(expirationTimestamp) && !_.isNil(createdTimestamp) && !_.isNil(autoRenewPeriod)
+    ? nsToSecNs(BigInt(createdTimestamp) + BigInt(autoRenewPeriod * constants.AUTO_RENEW_PERIOD_MULTIPLE))
```

Comment:
The database values are already bigint so just need to store the constant as BigInt as well and can avoid the extra conversion.
```suggestion
    ? nsToSecNs(createdTimestamp + autoRenewPeriod * constants.AUTO_RENEW_PERIOD_MULTIPLE)
```

Reply:
- I found that this part throws an error (even with the constant being a `BigInt`):  
   `autoRenewPeriod * constants.AUTO_RENEW_PERIOD_MULTIPLE` -> `Cannot mix BigInt and other types, use explicit conversions`
    The error can be resolved by casting `autoRenewPeriod` as a `BigInt`. 

- There are many spec tests that have a created timestamp of `987654111123456`, which doesn't enter the function as a BigInt, so it seems that some created timestamps need to be converted to `BigInt`. I've added this timestamp to the unit test.